relational_database/homework.py

```diff
@@ -44,7 +51,10 @@ def task_3_list_customers_in_germany(cur) -> list:
 
     Returns: 11 records
     """
-    pass
+    cur.execute("SELECT * FROM Customers "
+                "WHERE country = 'Germany';"
+                )
```

Comment:
it is bad way to move `)` as you. Read about formatting multiline strings, lists

Reply:
Thanks for your comments, fixed.